CHANGELOG.md

```diff
@@ -19,13 +19,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## @mercurial-finance/dynamic-amm-sdk [0.4.27] - PR #142
+
+### Added
+
+- `getPoolConfigWithPoolCreatorAuthority` to get pool config based on the given wallet
```

Comment:
should update the changelog for createPermissionlessPoolWithConfig only accept pool signer as config.pool_creator_authority?

Reply:
We still have config that can be used by anyone

Reply:
yes, I mean we should distinguish that config.pool_creator_authority == Pubkey::default() then anyone can use this, otherwise only  pool_creator_authority can create the pool with that config key

Reply:
Fixed